blocks/api/post.pegjs

```diff
@@ -95,7 +112,7 @@ WP_Block_Balanced
   }
 
 WP_Block_Html
-  = ts:(!WP_Block_Balanced !WP_Block_Void c:Any {
+  = ts:(!WP_Block_Balanced !WP_Block_Void !WP_Block_NextPage c:Any {
```

Comment:
Should probably also put in `!WP_Tag_More` here until we fix up the root cause of this trouble